dcos/subcommand.py

```diff
@@ -118,6 +124,30 @@ def distributions():
         return []
 
 
+def command_info(command):
+    """top level documentation of default DCOS-CLI command
+    :param command: name of command
+    :param type: str
+    :returns: command summary
+    :rtype: str
+    """
+
+    return command_documentation(command).split('\n')[0]
```

Comment:
I'm not sure about adding these methods to `SubcommandMain'. That class just runs a main method in a thread, and doesn't know about the internals of the running method. It also takes command args as the constructor which wouldn't make sense outside that context.